### DISCUSSION TITLE
Google Search Console integration, only works for 1 site

### DISCUSSION THREAD

**Discussion by darrylmorley** [plausible/analytics#4303] [Self-Hosted Support]
Hi,

How can I enable Google Search Console Integration for all of my sites? 

Whenever I try to connect a new site to Google Search Console I get an error: "Access blocked: This app’s request is invalid".

I have 5 sites in total at the moment, and only 1 of them works with Search Console.

Grateful if anyone can point me in the right direction please.

**Comment by ruslandoga** [comment]
👋 @darrylmorley 

Quick check, have you tried the guide at https://github.com/plausible/community-edition#google-integration?

**Reply by darrylmorley** [reply]
> 👋 @darrylmorley 
> 
> 
> 
> Quick check, have you tried the guide at https://github.com/plausible/community-edition#google-integration?

Yeah, that's how I got my initial site setup. Worked great, but now can't link any other sites.

**Reply by darrylmorley** [reply]
Nevermind, completely my fault. Sorry. I changed the plausible URL and forgot to update the Google Client. 😱
